addons/account/tests/common.py

```diff
@@ -285,6 +285,12 @@ def setup_multi_currency_data(cls, default_values=None, rate2016=3.0, rate2017=2
             'currency_subunit_label': 'Silver',
             **default_values,
         })
+        rate0 = cls.env['res.currency.rate'].create({
+            'name': '1900-01-01',
+            'rate': 1,
+            'currency_id': foreign_currency.id,
+            'company_id': cls.env.company.id,
+        })
```

Comment:
Not sure if it's better to provide this in the setup or to update every test that was relying on the previous behavior for it's setup and assertion values